### PR TITLE
Added blocking user code sample

### PR DESCRIPTION
**Pull request by silkypriya**
Based on #53 
I added sample code to block a user to demonstrate the use of API:Block.
The link of this action API module is : https://www.mediawiki.org/wiki/API:Block

### CODE REVIEW

**Comment by srish** [comment]
@silkypriya Thanks for your contribution to this project! But, this is not related to any microtask that I've added to the task description on Phabricator [T215682](https://phabricator.wikimedia.org/T215682) :-/ 

I will try to create a microtask around the `Options` module and then will get back to you in a day or so! 

In the future, before starting work on a topic /  sending a pull request, I would appreciate it if you discuss it with me on Zulip or Phabricator. That way I can verify if the particular contribution would be helpful or not for the project at the moment.

Same comment for #57

**Comment by silkypriya** [comment]
@srish I apologize for this,will take care in future. Thank you.

**Comment by srish** [comment]
I've added your Phabricator username next to `API:Emailuser` and `API:Block` module in the task description https://phabricator.wikimedia.org/T215682. I will be reviewing your pull requests soon!

File: python/blocking_user.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+R = S.get(url=URL, params=PARAMS_2)
+DATA = R.json()
+
+CSRF_TOKEN = DATA['query']['tokens']['csrftoken']
+
+# Step 4: POST request to block user
+PARAMS_3 = {
+    'action':"block",
+    'user':"user_name",
+    'expiry':"expiry_time",
```
Comment (srish) [inline python/blocking_user.py]: You can add a value for time to explain what the format would be.

**Comment by silkypriya** [comment]
Thank you! @srish for assigning documentation task for `API:Emailuser` and `API:Block` module.
I am working on API Documentation page of these modules.I will submit the API documentation pages and final changes in code by tomorrow.
Thank you for your feedback and suggested improvements :D

**Comment by silkypriya** [comment]
Hello @srish , thank you for suggested improvements.I made commit after Pylint test and adding value for time and reason same as POST request example given in `API:Block` documentation.I also added the code for unblocking a user.
Would love your feedback.

**Comment by srish** [comment]
@silkypriya Thanks for your work on this! Here is first round of feedback: https://www.mediawiki.org/wiki/Topic:Uv9ne6b7svs6rrnh :)

**Comment by silkypriya** [comment]
Thank you @srish for feedback and suggested improvements. I made the changes and this is the link (https://www.mediawiki.org/wiki/User:Silky44/Sandbox/API:Block) of the modified document. 

1. In the "Example" section, explained steps to complete the POST request like in referred example and 
    put the extra content in notes. Or shall I include the extra content in "Additional notes".
2. Combined the possible errors sections.Shall I remove the common possible errors which are common to both blocking and unblocking user.
 
Please let me know,if more changes are required.
Thank you!

**Comment by srish** [comment]
@silkypriya Sweet! thanks for your work on this! I've made a few minor changes here: https://www.mediawiki.org/wiki/User:Silky44/Sandbox/API:Block, mostly added a "Parameter history" section and then while reviewing code removed the parameters you've added for this file as that is not required. That file is only used to autogenerate python files for modules that support GET requests. 

You can copy paste the changes from your Sandbox to main namespace on MediaWiki.org and leave a comment while publishing changes like: "updated as per documentation improvements project Phab:T198916"

**Comment by silkypriya** [comment]
Thank you so much @srish for merging this pull request and the improvements!

**Pull request merged**